### ISSUE TITLE
CLI does not build properly

### ISSUE THREAD

**Issue by stesix** [Cretezy/MasterPassX#11]
It can't find the module 'masterpassx-core' and a bunch of others. Even by creating a fake link in the node_modules does not achieve much.

**Comment by stesix** [comment]
problem found: the cli references the wrong version of the masterpassx-core.

**Comment by Cretezy** [comment]
Are you sure about this? https://www.npmjs.com/package/masterpassx-core shows the published version at 1.1.1.

**Comment by stesix** [comment]
Nope... that was not the real problem or better it solved the issue for a while. Currently I think there's something funny with my yarn.

Now I'm getting the following fantastic errors:
```
src/index.js → dist/index.js...
(!) Unresolved dependencies
https://github.com/rollup/rollup/wiki/Troubleshooting#treating-module-as-external-dependency
events (imported by node_modules/commander/index.js, commonjs-external:events, node_modules/signal-exit/index.js)
child_process (imported by node_modules/commander/index.js, commonjs-external:child_process, node_modules/external-editor/main/index.js, node_modules/execa/index.js, node_modules/cross-spawn/index.js, ../node_modules/spawn-sync/index.js, ../node_modules/spawn-sync/lib/spawn-sync.js, node_modules/arch/index.js)
path (imported by node_modules/commander/index.js, commonjs-external:path, src/index.js, node_modules/tmp/lib/tmp.js, node_modules/cross-spawn/lib/util/resolveCommand.js, node_modules/which/which.js, ../node_modules/spawn-sync/lib/spawn-sync.js, node_modules/npm-run-path/index.js, node_modules/clipboardy/lib/linux.js, node_modules/clipboardy/lib/windows.js, node_modules/arch/index.js)
fs (imported by node_modules/commander/index.js, commonjs-external:fs, src/index.js, node_modules/external-editor/main/index.js, node_modules/tmp/lib/tmp.js, node_modules/chardet/index.js, node_modules/isexe/index.js, node_modules/isexe/windows.js, node_modules/isexe/mode.js, node_modules/cross-spawn/lib/util/readShebang.js, ../node_modules/spawn-sync/lib/spawn-sync.js, node_modules/arch/index.js)
os (imported by src/index.js, node_modules/supports-color/index.js, commonjs-external:os, ../node_modules/spawn-sync/lib/spawn-sync.js, ../node_modules/os-shim/lib/os.js)
util (imported by src/index.js, node_modules/chardet/encoding/unicode.js, commonjs-external:util, node_modules/chardet/encoding/mbcs.js, node_modules/chardet/encoding/sbcs.js, node_modules/chardet/encoding/iso2022.js, node_modules/execa/index.js, node_modules/lru-cache/index.js)
stream (imported by node_modules/through/index.js, commonjs-external:stream, node_modules/mute-stream/mute.js, node_modules/iconv-lite/lib/streams.js, node_modules/iconv-lite/lib/extend-node.js, node_modules/get-stream/buffer-stream.js)
readline (imported by node_modules/inquirer/lib/ui/baseUI.js, commonjs-external:readline)
assert (imported by node_modules/signal-exit/index.js, commonjs-external:assert, node_modules/inquirer/lib/objects/choices.js)
tty (imported by node_modules/cli-width/index.js, commonjs-external:tty)
crypto (imported by node_modules/tmp/lib/tmp.js, commonjs-external:crypto)
buffer (imported by node_modules/iconv-lite/lib/index.js, node_modules/iconv-lite/encodings/internal.js, commonjs-external:buffer, node_modules/iconv-lite/encodings/utf16.js, node_modules/iconv-lite/encodings/utf7.js, node_modules/iconv-lite/encodings/sbcs-codec.js, node_modules/iconv-lite/encodings/dbcs-codec.js, node_modules/iconv-lite/lib/streams.js, node_modules/iconv-lite/lib/extend-node.js)
string_decoder (imported by node_modules/iconv-lite/encodings/internal.js, commonjs-external:string_decoder)
try-thread-sleep (imported by ../node_modules/spawn-sync/lib/spawn-sync.js, commonjs-external:try-thread-sleep)
created dist/index.js in 6s
```

I've tried to obliterate every node_modules folder in the project and to re-execute the whole `yarn` from the project root and `yarn build` from each project but it still doesn't want to work.

If you've got some suggestions, I'm very happy to hear.
